app/qml/inputs/MMPhotoEditor.qml

```diff
@@ -0,0 +1,67 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Controls.Basic
+import "../components"
+
+MMAbstractEditor {
+  id: root
+
+  property var parentValue: parent.value ?? ""
+  property bool parentValueIsNull: parent.valueIsNull ?? false
+  property bool isReadOnly: parent.readOnly ?? false
+
+  property url photoUrl
+
+  signal trashClicked()
+
+  contentItemHeight: 160 * __dp
+  spacing: 0
+  radius: 20 * __dp
+
+  content: MMPhoto {
```

Comment:
Sooo what if there is no photo added yet? :) There is an updated in the design system. We can add it in a different PR though

Reply:
MMPhoto contains already an icon, when photo is not loaded by the design system